tests/lowering/eltwise/unary/test_bitwise_not.py

```diff
@@ -0,0 +1,34 @@
+import torch
+import torch_ttnn
+import pytest
+import ttnn
+
+
+class BitwiseNotModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, input):
+        return torch.bitwise_not(input)
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    [(4, 4)],
+)
+def test_bitwise_not(device, input_shape):
+    m = BitwiseNotModule()
+    input = torch.randint(-256, 256, input_shape, dtype=torch.int32)
```

Comment:
Here is a problem.  I make this tensor in `torch.int32`, but the test complains that it gets a BFLOAT16 tensor.
```
E       RuntimeError: TT_FATAL @ /home/jdh8/tt-metal/ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_device_operation.cpp:42: input_datatype == DataType::INT32
E       info:
E       Unsupported input data type '0' for UnaryOpType '63' (Bitwise operation).
E       backtrace:
E        --- /home/jdh8/tt-metal/ttnn/ttnn/_ttnn.so(+0x4933a9) [0x7f0db03ee3a9]
E        --- ttnn::operations::unary::UnaryDeviceOperation::validate_on_program_cache_miss(ttnn::operations::unary::operation_attributes_t const&, ttnn::operations::unary::tensor_args_t const&)
```

Previous error message also confirms this.

tenstorrent/tt-metal#12902 does not fix this error.

Reply:
@kevinwuTT ptal

Reply:
This error still persists after merging tenstorrent/tt-metal#13579.

Reply:
@pschang-phy, this was my failed attempt to lower `aten.bitwise_not`.  I just confirmed that this error still persists.  I hope this is helpful for your work on #634.

Reply:
Here is my try. It seems need to explicitly cast to **TtnnInt32**
```
cast_to_int32 = g.call_function(ttnn.typecast, args=(args[0], TtnnInt32()))
return g.call_function(ttnn.bitwise_not, args=(cast_to_int32,))
```